cra-rxjs-styled-components/src/components/issue/IssueTabHeader/IssueTabHeader.tsx

```diff
@@ -0,0 +1,85 @@
+import { useCallback, useState } from 'react';
+import DetailsDropdown from '../../DetailsDropdown';
+import CorrectIcon from '../../icons/CorrectIcon';
+import IssueIcon from '../../icons/IssueIcon';
+import type { IssueTabValues } from '../types';
+
+import { Container, StatusLabel, StatusTab } from './IssueTabHeader.style';
+import type { DropdownTitle } from '../types';
+
+interface Props {
+  toggleTab: any;
+}
+
+export default function IssueTabHeader({ toggleTab }: Props) {
+  const [activeTab, setActiveTab] = useState<IssueTabValues>('open');
+  const [openDropdown, setOpenDropdown] = useState<DropdownTitle | undefined>();
+
+  const changeTab = (value: IssueTabValues) => {
+    toggleTab(value);
+    setActiveTab(value);
+  };
+
+  const toggleDropdown = useCallback(
+    (dropdownName: DropdownTitle) => {
+      setOpenDropdown(openDropdown === dropdownName ? undefined : dropdownName);
```

Comment:
a better approach when comes to modifying the prev state value on the state, is doing: 
```javascript
setOpenDropdown((prevValue) => prevValue === dropdownName ? undefined : dropdownName))
```

Avoiding async problems!